### PR TITLE
Update database at startup

### PR DESCRIPTION
**Pull request by vpavlin**
## Related Issues and Dependencies

Closes https://github.com/opendatahub-io/odh-manifests/issues/311

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Run JH upgrade-db on start

### CODE REVIEW

File: start-jupyterhub.sh [hunk 1]
```diff
@@ -12,6 +12,10 @@ PATH=$PATH:/opt/app-root/scripts
 
 . /opt/app-root/etc/jupyterhub_config.sh
 
+# Upgrade DB is needed
```
[the remark this answers is not in the document]
Comment (vpavlin) [reply start-jupyterhub.sh]: Lol...I fixed both of these right after I submitted the PR, but did not push....push done now:)

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by 4n4nd — APPROVED** [review]
/llgtm

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**